IS7024FinalProjectBreweryArc/BreweryArc/Pages/Shared/_Layout.cshtml

```diff
@@ -18,9 +18,9 @@
                 </button>
                 <div class="navbar-collapse collapse d-sm-inline-flex flex-sm-row-reverse">
                     <ul class="navbar-nav flex-grow-1">
-                        <li class="nav-item">
-                            <a class="nav-link text-dark" asp-area="" asp-page="/Index">Home</a>
-                        </li>
+                        @*<li class="nav-item">

```

Comment:
When I execute the project in the home screen BrewaryArc, Home and Logo buttons contains same functionality. So, I have removed the Home button on Layout page.
https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions

Reply:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.